packages/ui/src/components/atoms/Input/InputLabel.tsx

```diff
@@ -1,32 +1,90 @@
-import React, { FC, ReactNode } from 'react';
-import styled from 'styled-components';
+import React, { FC, ReactNode, LabelHTMLAttributes } from 'react';
+import styled, { css } from 'styled-components';
 
-interface InputLabelProps {
+import {
+  spacing,
+  SpacingProps,
+  BorderProps,
+  border,
+  font,
+  display,
+  DisplayProps,
+  FontProps,
+} from '../../utils';
+
+interface InputLabelProps
+  extends LabelHTMLAttributes<HTMLLabelElement>,
+    SpacingProps,
+    BorderProps,
+    FontProps,
+    DisplayProps {
   children?: ReactNode;
+  className?: string;
+  fontSize?: number;
+  fontWeight?: FontProps['$fontWeight'];
+  textAlign?: 'left' | 'right';
+  noWrap?: boolean;
+  clickable?: boolean;
+  inline?: boolean;
+  color?: string;
 }
 
-const InputLabelStyle = styled.label`
-  display: block;
-  text-align: left;
+const InputLabelStyle = styled.label<InputLabelProps>`
+  ${font} // Utilize the font utility
+  ${display} // Utilize the display utility
+  text-align: right;
   width: 100%;
 
-  display: block;
-  text-align: left;
-  width: 100%;
+  color: ${({ theme, color }) => color ?? theme.palette.text.muted};
+  ${({ textAlign }) => textAlign === 'left' && 'text-align: left;'}
+  white-space: ${({ noWrap }) => (noWrap ? 'nowrap' : 'normal')};
+  ${spacing}
+  ${border}
 
-  font-size: 14px;
-  font-weight: 300;
-  letter-spacing: 0.12em;
```

Comment:
Do not modify base css of existing components

Reply:
done

Reply:
Its still modified